lib/camp.js

```diff
@@ -745,25 +745,45 @@ function Path(matcher) {
 }
 
 // Path unit.
-function pathLikeUnit(serverMethod, statusCode) {
-  statusCode = statusCode || 200;
+function pathLikeUnit(serverMethod, statusCode, httpMethods) {
+  httpMethods = httpMethods || [];
   return function unit(server) {
-    var paths = [];
     var callbacks = [];
 
-    var addPath = function (path, callback) {
-      paths.push(new Path(path));
-      callbacks.push(callback);
+    var addCallback = function (path, callback) {
+      callbacks.push({
+        path: new Path(path),
+        methods: null,
+        callback: callback
+      });
     };
+    server[serverMethod] = addCallback;
+
+    // Method-specific helpers like .get(), .post(), etc.
+    httpMethods.forEach(function (method) {
+      server[method.toLowerCase()] = function (path, callback) {
+        callbacks.push({
+          path: new Path(path),
+          methods: [method],
+          callback: callback
+        });
+      };
+    });
 
-    server[serverMethod] = addPath;
     return function pathLayer (req, res, next) {
-      var pathLen = paths.length;
+      var pathLen = callbacks.length;
       var matched = null;
       var cbindex = -1;
       for (var i = 0; i < pathLen; i++) {
-        matched = paths[i].match(req.path);
-        if (matched !== null) { cbindex = i; break; }
+        matched = callbacks[i].path.match(req.path);
+        if (!matched) {
```

Comment:
If possible, I would prefer to have booleans, instead of implicit conversion, in statement conditions.